server/server.ts

```diff
@@ -96,6 +97,16 @@ io.on("connection", (socket) => {
     // Send GAME_CREATED event
     io.to(host.socketId).emit("GAME_CREATED", gameCode);
   });
+
+  socket.on("ALLOCATE_PLAYERS", async (gameCode) => {
+    const game: Game | undefined = gamesMap.get(gameCode);
+    if (!game) {
+      return;
+    }
+    const players = game.getPlayers();
```

Comment:
Maybe make it so handleRoomAllocation can be called from the game class. Would rather not expose information when possible